src/Host.cpp

```diff
@@ -178,10 +178,11 @@ Host::Host(int port, const QString& hostname, const QString& login, const QStrin
     // file auditing and other issues)
     mErrorLogStream.setDevice(&mErrorLogFile);
 
-    // There was a map load attempt made here but it did not seem to be needed
-    // and it caused issues being doing in the constructor (some other classes
-    // were not fully initialised at this point) so it seemed sensible to remove
-    // it - Slysven
```

Comment:
When I said "...it did not seem to be needed..."  that partly covered the fact that (I think if I recall correctly) that either the "default host" was trying to load a map or an existing map was loaded ___twice___ (or possibly both)!